packages/realm-web/src/App.ts

```diff
@@ -44,6 +45,9 @@ export class App<
     /** @inheritdoc */
     public readonly services: Realm.Services;
 
+    /** @inheritdoc */
+    public readonly auth: Realm.AuthProviders;
```

Comment:
use `authProviders` instead of just `auth` ?

Reply:
This name was agreed on in the "Stitch/Realm SDK Merge And Naming" meeting. If you feel strongly for this, we could reopen the conversation in the #realm-sdk-team channel?

Reply:
its a naming discrepancy in my view. since we named `Realm.Services` as `services` and then we rename `Realm.AuthProviders` as `auth`. But if it's something that was discussed then lets leave it as it is.